pkgs/applications/office/timekeeper/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, poco
+, pkgconfig
+, gnome2
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "timekeeper";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "bburdette";
+    repo = "TimeKeeper";
+    rev = "363a3f0033fcca2d45b0d1f0fc74d6470c9db10d";
+    fetchSubmodules = true;
+    sha256 = "03alrfy7axq64k9lrs14afc0m6zn5gr5glhgx9i2r1yam7wdmz5c";
+  };
+
+  buildInputs = [
+    poco
+    pkgconfig
+  ] ++ (with gnome2; [
+    gtkmm
+    libglademm
+    libglade
+  ]);
+
+  NIX_CFLAGS_COMPILE = "-isystem ${lib.getDev gnome2.libglade}/include/libglade-2.0";
```

Comment:
Can you fix the program instead to find the headers using pkg-config?

Reply:
fixed!